object.go

```diff
@@ -107,7 +107,9 @@ func (d *Date) UnmarshalText(data []byte) error {
 type FileType string
 
 type File struct {
```

Comment:
According to [this](https://developers.notion.com/reference/page#files-property-values) documentation the `File` struct contains the `external` property also